microservices/site-launch/lambda-functions/redirection-domain-validation/index.ts

```diff
@@ -41,6 +43,23 @@ export const redirectionDomainValidation = async (
     auth: process.env.SYSTEM_GITHUB_TOKEN,
   })
 
+  let fileExists = true
+  // see if domain commit in github first. If exists, dont create commit.
+  try {
+    await octokit.request(
```

Comment:
Hmm I actually rather not for this case. For the lambds, I can't have any reference to any modules within our monolith codebase, because then we will be deploying the entire node-modules over to the lambdas, which would be too much. 
Since this boilerplate small enough code, and we dont expect this code to change in the future, I propose to keep it as it is, wdyt?

Reply:
hmm yea sure we can leave it as-is